### PR TITLE
Initial update for January 2020 release

### PR DESCRIPTION
**Pull request by joshfree**
Update old November and December references to January.

A second PR will be needed to update all links to the latest README.md Tags next week

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This repository is intended for active development of the Azure SDK for Java. Fo
 
 ### Important
 
-The Azure SDK team is pleased to make available the November 2019 client library General Availability (GA) release. We strongly recommend using the GA libraries in all production environments as these libraries are well tested and officially supported by Microsoft. More details, including installation instructions can be found here [here](https://azure.github.io/azure-sdk/releases/2019-11/java.html).
+The Azure SDK team is pleased to make available the January 2020 client library General Availability (GA) release. We strongly recommend using the GA libraries in all production environments as these libraries are well tested and officially supported by Microsoft. More details, including installation instructions can be found here [here](https://azure.github.io/azure-sdk/releases/2019-11/java.html).
```
Comment (samvaity) [inline README.md]: Will this link also be updated in the upcoming PR?

Comment (joshfree) [reply README.md]: good catch, yes; I had only been thinking about the links to (yet to be created) release tags

File: README.md [hunk 2]
```diff
@@ -16,24 +16,28 @@ For tutorials, samples, quick starts and other documentation, visit [Azure for J
...
 
 These libraries can be easily identified by sdk/ folder, package, and namespaces names starting with `azure-`, e.g. `azure-keyvault`.
 
-The libraries released in the GA December 2019 release:
+The libraries released in the GA January 2020 release:
+- [App Configuration](https://github.com/Azure/azure-sdk-for-java/tree/azure-data-appconfiguration_1.0.0-beta.7/sdk/appconfiguration/azure-data-appconfiguration/README.md)
+- [Event Hubs](https://github.com/Azure/azure-sdk-for-java/blob/azure-messaging-eventhubs_5.0.0-beta.6/sdk/eventhubs/azure-messaging-eventhubs/README.md)
+- [Event Hubs Checkpoint Store](https://github.com/Azure/azure-sdk-for-java/blob/azure-messaging-eventhubs-checkpointstore-blob_1.0.0-beta.4/sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md)
 - [Identity](https://github.com/Azure/azure-sdk-for-java/blob/azure-identity_1.0.1/sdk/identity/azure-identity/README.md)
+- [Key Vault Certificates](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-certificates_4.0.0-beta.6/sdk/keyvault/azure-security-keyvault-certificates/README.md)
```
Comment (g2vinay) [inline README.md]: These links are not correct,
But I believe we will be updating them in follow up PR

Comment (joshfree) [reply README.md]: That's correct.  They're more than 1 release behind in fact.

Once the January 2020 release tags are created, we can then update the readme to reference them wholesale.

**Pull request merged**